src/transactions/utils/validation.js

```diff
@@ -28,8 +28,18 @@ export const validatePublicKey = publicKey => {
 	return true;
 };
 
+export const checkPublicKeysForDuplicates = publicKeys =>
+	publicKeys.every((element, index) => {
+		const elementFound = publicKeys.indexOf(element);
```

Comment:
How about `publicKeys.slice(index + 1)`? Makes it slightly more efficient and enables a simpler `if` check below.